src/status_im/chat/events.cljs

```diff
@@ -120,38 +110,27 @@
                                  :show-emoji?       false
                                  :bottom-info       details})))
 
+(def ^:private index-messages (partial into {} (map (juxt :message-id identity))))
+
 (handlers/register-handler-fx
   :load-more-messages
   [(re-frame/inject-cofx :get-stored-messages)]
-  (fn [{{:keys [current-chat-id loading-allowed] :as db} :db
+  (fn [{{:keys [current-chat-id] :as db} :db
         get-stored-messages :get-stored-messages} _]
     (let [all-loaded? (get-in db [:chats current-chat-id :all-loaded?])]
```

Comment:
`all-loaded?` is not used anywhere except next `when-not` form so we probably can just skip this `let`

Reply:
also it's a bit weird that we later shadow it with another `all-loaded?`

Reply:
Correct, that shadowing is ugly, I didn't realise it, good catch !
I will fix it